templates/web/base/tokens/confirm_update.html

```diff
@@ -1,24 +1,24 @@
 [% INCLUDE 'header.html', bodyclass = 'fullwidthpage', title => loc('Confirmation') %]
 
-<div class="confirmation-header">
-  <h1>[% loc('Your update has been confirmed!') %]</h1>
-
-<p class="confirmed">
 [% IF creator_fixed %]
-[%
-    tprintf(loc('Thank you &mdash; you can <a href="%s">view your updated problem</a> on the site.'),
-        c.uri_for( '/report', problem_id ) );
-%]
+  [% problem_url = c.uri_for('/report', problem_id) %]
 [% ELSE %]
-[%
-    tprintf( loc('You have successfully confirmed your update and you can now <a href="%s">view it on the site</a>.'),
-        c.uri_for( '/report', update.problem.id ) _ '#update_' _ update.id );
-%]
+  [% problem_url = c.uri_for( '/report', update.problem.id ) _ '#update_' _ update.id %]
 [% END %]
-</p>
+
+<div class="confirmation-header">
+
+  <h1><a href="[% problem_url %]">[% update.problem.title %]</a></h1>
+
+  <h2>[% loc('Thank you for updating this issue!') %]</h2>
```

Comment:
The difference in page widths between `creator_fixed.html` and `confirm_update.html` is a bit odd, but I don't think it's a big deal, and this pull request has already sprawled far enough from the original issue. Feel free to correct it if you feel strongly otherwise.